code/game/objects/items/circuitboards/machines/machine_circuitboards.dm

```diff
@@ -300,6 +300,15 @@
 		/obj/item/stack/cable_coil = 1,
 		/obj/item/stack/sheet/glass = 1)
 
+/obj/item/circuitboard/machine/thermomachine/anomalous
+	name = "Anomalous Heating Thermomachine"
+	build_path = /obj/machinery/atmospherics/components/unary/thermomachine/anomalous
+	req_components = list(
+		/datum/stock_part/matter_bin = 2,
+		/obj/item/assembly/signaler/anomaly/pyro = 1,
+		/obj/item/stack/cable_coil = 1,
+		/obj/item/stack/sheet/plasteel = 1)
```

Comment:
```suggestion
		/obj/item/stack/sheet/plasteel = 1,
	)
```

i know theyre all like this. they shouldnt be

Reply:
Gotcha, adjusted